src/convertToObject.js

```diff
@@ -3,16 +3,34 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ *  (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const separateProperties = sourceString
+    .split(';');
+
+  const separateKeysValues = separateProperties
+    .map(el => el.trim().split(':'));
+
+  const convertObject = separateKeysValues.reduce((obj, currentValue) => {
```

Comment:
Can we name it as 'styles'?

Reply:
Fix this. Also try to make better name than `obj`